src/module_loader.rs

```diff
@@ -23,6 +25,7 @@ struct InnerRustyLoader {
     source_map_cache: Rc<RefCell<SourceMapCache>>,
 }
 
+#[allow(dead_code)]

```

Comment:
This was to silence a warning about one of the methods in that block that only happened when fs_import was enabled in rust-analyzer, because the whitelist is only being checked when fs imports are disabled.